### ISSUE TITLE
[fix] Language selector for two languages has invalid ARIA attribute

### ISSUE THREAD

**Issue by jaredcunha** [trussworks/react-uswds#2809]
**ReactUSWDS Version & USWDS Version:**

There on the two-language variants of the Language Selector (button or link), there is an accessibility defect coming from `aria-controls="language-options"`. An element with `id="language-options"` would not need to exist for this control.

While `aria-controls` might not be function, this is showing up as a violation in automated tooling.

**To Reproduce**

1. Click on the language selector button (or link)
2. Inspect element
3. Note the `aria-controls` attribute

**Expected behavior**

The `aria-controls` attribute should not be on this version of the Language Selector

**Screenshots**

<img width="556" alt="Screenshot 2024-03-08 at 11 20 23 AM" src="https://github.com/trussworks/react-uswds/assets/1094951/fe0dec95-05f7-4572-8674-a33da7f40cd2">

**Additional context**

**Device and Browser Information (please complete the following information if describing a UI bug):**
